src/main/java/org/mskcc/cbio/oncokb/domain/TokenStats.java

```diff
@@ -141,4 +141,17 @@ public String toString() {
             ", usageCount=" + getUsageCount() +
             "}";
     }
+
+    public String toCSV() {
+        return getId() +
+            ";" + getAccessIp() +
+            ";" + getResource() +
+            ";" + getAccessTime() +
+            ";" + getUsageCount() +
+            ";" + getToken().getId();
+    }
+
+    public static String csvColumns() {
+        return "id;accessIp;resource;accessTime;tokenId;usageCount";
```

Comment:
Id is not necessary here anymore. We need user email instead for further association.

Reply:
@zhx828 On this note, would it be more useful to sort the entries by email or accessTime? I will do accessTime for now.